src/resources/snippets.ts

```diff
@@ -204,6 +204,10 @@ export default [
     "preview": "resources/assets/snippets/pretty-lyrics.png",
   },
   {
+    "title": "Oneko",
+    "description": "Adds Oneko onto your playback bar!",
+    "code": "[class^='playback-']:not(.volume-bar__slider-container > .playback-progressbar):before { content: ''; width: 32px; height: 32px; bottom: calc(100% - 7px); right: 10px; position: absolute; image-rendering: pixelated; background-image: url('https://raw.githubusercontent.com/adryd325/oneko.js/14bab15a755d0e35cd4ae19c931d96d306f99f42/oneko.gif'); animation: oneko 1s infinite; } @keyframes oneko { 0%, 50% { background-position: -64px 0; } 50.0001%, 100% { background-position: -64px -32px; } }",
+    "preview": "resources/assets/snippets/oneko.png"
     "title": "Remove Popular sections from homepage",
```

Comment:
It looks like I messed it up a bit in the merge. I think this should fix it?

```suggestion
    "preview": "resources/assets/snippets/oneko.png",
  },
  {
    "title": "Remove Popular sections from homepage",
```